src/cordova/setup.action.mjs

```diff
@@ -62,6 +62,12 @@ export async function main(...parameters) {
   if (isApple) {
     // since apple can only be build on darwin systems, we don't have to worry about windows support here
     // TODO(daniellacosse): move this to a cordova hook
+    execSync(`cd third_party/CocoaLumberjack && make clean && make`, {
```

Comment:
Are we building the libraries from source?

Reply:
yeah carthage pulls from github